Add second Copilot video and more cleanup (#4160)
SumoLogic/sumologic-documentation: docs/search/copilot.md

id: copilot
title: Sumo Logic Copilot - Feature Preview
sidebar_label: Copilot 🤖

> description: Streamline your log analysis with Sumo Logic Copilot, our AI-based assistant designed to simplify log analysis by allowing you to ask questions in plain English and providing search suggestions without the need to write log queries.

Reviewer: ```suggestion
description: Streamline your log analysis with Sumo Logic Copilot, our AI-based assistant. Copilot allows you to ask questions in plain English and provides search suggestions, all without the need to write log queries.
```

import Iframe from 'react-iframe';
import useBaseUrl from '@docusaurus/useBaseUrl';

<p><a href="/docs/beta"><span className="beta">Preview Release</span></a></p>
This is a Preview release. To learn more, contact your Sumo Logic account executive.

> Sumo Logic Copilot is an AI-based assistant designed to simplify log analysis by allowing you to ask questions in plain English and providing search suggestions without the need to write log queries.

Reviewer: ```suggestion
Sumo Logic Copilot is an AI-based assistant designed to simplify log analysis. It allows you to ask questions in plain English and provides search suggestions, all without the need to write log queries.
```

1. Add the security `groupid` to the prompt to tabulate events containing that data.
1. Focus on a specific access key, `ABCDEFGOYCM3PIKNOVRA`, noting `PutRolePolicy` events indicating permission elevation. Multiple AWS accounts are impacted.
1. We conclude that the access key was used to modify permissions, providing the attacker with a potential network entry point. Remediation would involve disabling the key, blocking the source IP, and further log analysis.

> ### Security example

Reviewer: I'm confused why you are including this video. It is the exact same example as the previous one, but presented without audio, and with visual highlighting of the steps.

Author: Same here! I've asked Bashyam about this in Slack.

Author: I've merged this branch (`copilot-vid`) into the `ai-machine-learning` branch. I'll work with Bashyam to sort out the duplicate video issue in [this PR](https://github.com/SumoLogic/sumologic-documentation/pull/3793).
